src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
```

Comment:
[Redundant empty line ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#please-dont-add-redundant-empty-lines-to-your-code)